src/components/Types.ts

```diff
@@ -9,6 +9,19 @@ export type Drink = {
   idDrink: string;
   strDrink: string;
   strDrinkThumb: string;
+  strInstructions?: string | null;
```

Comment:
We'll need to see if we can get the "extends" working either in the type or as an interface